examples/db.yaml

```
@@ -13,31 +13,31 @@ imports: |
     print("Listing all databases")
-  view|v: |
-    """View database table"""
-    console.print(f"Viewing {table} table for {name} DB")
-
-
-args:
-  create: &db_args
-    - name: DatabaseName
-  delete: *db_args
-  view: 
+  view|v: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)